### PR TITLE
Updates for release v3.10

### PR DESCRIPTION
**Pull request by lmm**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: _includes/v3.10/release-notes/v3.10.0-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+18 Oct 2019
+
+#### Support top-level namespace and service account network policy selectors
+
+Calico now supports 2 new top-level selectors to make writing Calico network policy easier. The `namespaceSelector` allows you to select the namespace(s) to apply a global network policy to.
```
Comment (caseydavenport) [inline _includes/v3.10/release-notes/v3.10.0-release-notes.md]: "2" -> "two"

File: _includes/v3.10/release-notes/v3.10.0-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+#### Bug fixes
+
+ - Fix returning of resp.Body.Close() error. [node #329](https://github.com/projectcalico/node/pull/329) (@xieyanker)
+ - Fix erroneous "Partial success" output. [calicoctl #2070](https://github.com/projectcalico/calicoctl/pull/2070) (@lmm)
+ - Fix logging to be independent of go mod versions. [libcalico-go #1127](https://github.com/projectcalico/libcalico-go/pull/1127) (@neiljerram)
+
+#### Other changes
+
+ - Update calico/node to use a version of calico/bird based on upstream BIRD v1.6.7. This allows us to make use of LLGR (long-lived graceful restart). [node #314](https://github.com/projectcalico/node/pull/314) (@neiljerram)
```
Comment (caseydavenport) [inline _includes/v3.10/release-notes/v3.10.0-release-notes.md]: I think we haven't enabled LLGR, so it's misleading to mention it here.

File: _includes/v3.10/release-notes/v3.10.0-release-notes.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+ - Use a real confd version. [node #315](https://github.com/projectcalico/node/pull/315) (@rafaelvanoni)
```
Comment (caseydavenport) [inline _includes/v3.10/release-notes/v3.10.0-release-notes.md]: I think this one can be ignored - it doens't have a user-facing impact.

File: _includes/v3.10/release-notes/v3.10.0-release-notes.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+ - Add semaphore config. [node #324](https://github.com/projectcalico/node/pull/324) (@neiljerram)
+ - The default IP pool block size is now configurable. Use the `CALICO_IPV4POOL_BLOCK_SIZE` and `CALICO_IPV6POOL_BLOCK_SIZE` environment variables to configure the default pool block size. [node #346](https://github.com/projectcalico/node/pull/346) (@halfcrazy)
+ - The `CALICO_IPV4POOL_NODE_SELECTOR` and `CALICO_IPV6POOL_NODE_SELECTOR` environment variables were added for configuring default pool node selectors. [node #346](https://github.com/projectcalico/node/pull/346) (@tmjd)
+ - Add calicoctl `patch` command similar to `kubectl patch`. [calicoctl #2054](https://github.com/projectcalico/calicoctl/pull/2054) (@syscll)
```
Comment (caseydavenport) [inline _includes/v3.10/release-notes/v3.10.0-release-notes.md]: We might want to highlight the `calicoctl patch` command , actually. I think that's an exciting one.

File: _includes/v3.10/release-notes/v3.10.0-release-notes.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+ - Re-enable golangci-lint linters and fix lint warnings. [felix #2130](https://github.com/projectcalico/felix/pull/2130) (@rafaelvanoni)
+ - Add support to watch stderr for FV containers. [felix #2137](https://github.com/projectcalico/felix/pull/2137)(@song-jiang)
+ - Reduce severity of BPF log levels. [felix #2144](https://github.com/projectcalico/felix/pull/2144) (@tmjd)
+ - Fix felix log level. [felix #2147](https://github.com/projectcalico/felix/pull/2147) (@neiljerram)
+ - Clean up code. [felix #2150](https://github.com/projectcalico/felix/pull/2150) (@beautytiger)
+ - Run typha as non-root user. [typha #312](https://github.com/projectcalico/typha/pull/312)
+ - Improve log for check command failing. [confd #271](https://github.com/projectcalico/confd/pull/271) (@neiljerram)
+ - Move some testutils functions to utils_linux.go. [cni-plugin #787](https://github.com/projectcalico/cni-plugin/pull/787) (@song-jiang)
+ - Use 'git diff' instead of 'git diff-index' in Makefile's git-commit target. [cni-plugin #788](https://github.com/projectcalico/cni-plugin/pull/788) (@rafaelvanoni)
```
Comment (caseydavenport) [inline _includes/v3.10/release-notes/v3.10.0-release-notes.md]: This and the one below aren't user-facing so we should probably remove from here.

File: _includes/v3.10/release-notes/v3.10.0-release-notes.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+ - Add assertion that CI builds do not result in dirty state. [cni-plugin #790](https://github.com/projectcalico/cni-plugin/pull/790) (@caseydavenport)
+ - Handle removing node for flannel migration controller. [kube-controllers #404](https://github.com/projectcalico/kube-controllers/pull/404) (@song-jiang)
+ - Add flannel migration FV. [kube-controllers #406](https://github.com/projectcalico/kube-controllers/pull/406) (@song-jiang)
+ - Use a real logrus version. [libcalico-go #1124](https://github.com/projectcalico/libcalico-go/pull/1124) (@rafaelvanoni)
```
Comment (caseydavenport) [inline _includes/v3.10/release-notes/v3.10.0-release-notes.md]: This one too - not user-facing.

File: _includes/v3.10/release-notes/v3.10.0-release-notes.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+ - Ignore Windows reserved addresses. [libcalico-go #1129](https://github.com/projectcalico/libcalico-go/pull/1129) (@caseydavenport)
+ - Use Kubernetes 1.16 in tests. [libcalico-go #1131](https://github.com/projectcalico/libcalico-go/pull/1131) (@caseydavenport)
+ - Fix code warnings. [libcalico-go #1132](https://github.com/projectcalico/libcalico-go/pull/1132) (@beautytiger)
```
Comment (caseydavenport) [inline _includes/v3.10/release-notes/v3.10.0-release-notes.md]: Might want to combine this one or move it closer to the other code cleanup one from beautytiger?

File: _includes/v3.10/release-notes/v3.10.0-release-notes.md (hunk 4 above)
Comment (lmm) [reply _includes/v3.10/release-notes/v3.10.0-release-notes.md]: Agreed, done

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit e707f41e5e1b41211ed656ea29fde36e2af43877

https://deploy-preview-2948--calico.netlify.com

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 666ffe5d77ee21972db89bc5b3d8eb656adffbf3

https://deploy-preview-2948--calico.netlify.com

File: _includes/v3.10/release-notes/v3.10.0-release-notes.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+Calico now supports two new top-level selectors to make writing Calico network policy easier. The `namespaceSelector` allows you to select the namespace(s) to apply a global network policy to.
+This enables you to write a single network policy applicable to one or more namespaces. The `serviceAccountSelector` allows you apply a network policy only to the endpoints which have the specified service accounts attached to them.
+This selector can be useful in writing network policy to target endpoints with specific service accounts.
+
+For more information please see the how-to documentation for [namespace selectors](https://docs.projectcalico.org/v3.10/security/namespace-policy) and [service account selectors](https://docs.projectcalico.org/v3.10/security/service-accounts).
+
+ - Support top-level namespaceSelector for global network policies and serviceAccountSelector for global and namespaced network policies [libcalico-go #1133](https://github.com/projectcalico/libcalico-go/pull/1133) (@rafaelvanoni)
+
+#### Advertisement Kubernetes Service ExternalIPs
```
Comment (caseydavenport) [inline _includes/v3.10/release-notes/v3.10.0-release-notes.md]: "Advertisement _of_"

Comment (lmm) [reply _includes/v3.10/release-notes/v3.10.0-release-notes.md]: Thanks!

**Pull request merged**